Some upgrades to the XML defaults syntax

bartgol opened the pull request:
After #2458 , we will (hopefully!) start to add documentation to our parameters. Since putting all on one line may be hard to visualize on some text editors, splitting the XML defaults file on multiple lines is probably a good idea. For instance, we may do something like

```
    <atm_log_level type="string"
                   valid_values="trace,debug,info,warn,error"
                   doc="Verbosity level for the atm logger">
      info
    </atm_log_level>
```
However, the current XML parsing would consider as parameter value the whole string between `>` and `</`, including spaces and newlines. Hence, it would cause an error of the form
```
ERROR: Invalid value '
      info
    ' for element 'atm_log_level'. Value not in the valid list ('['trace', 'debug', 'info', 'warn', 'error']')
```
This PR fixes this, by stripping space/newline chars inside the param values.

E3SM-Autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4231</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;bugfix;CIME;AT: Skip Stand-Alone Testing</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>c162b56f934b83a5300d6943387b16c1db25537d</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>b0039c2576ae9a67d22dd6c235876b9df651b35e</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4766</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;bugfix;CIME;AT: Skip Stand-Alone Testing</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>c162b56f934b83a5300d6943387b16c1db25537d</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>b0039c2576ae9a67d22dd6c235876b9df651b35e</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: bartgol/eamxx/buildnml-parsing-fix</li>
    <li>SHA: c162b56f934b83a5300d6943387b16c1db25537d</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: bartgol
</details>

bartgol commented:
@jgfouca I changed the PR name, since I added another feature. Namely, we no longer need to provide "special" null values for arrays (so that the yaml parser can figure out the type of the array items). For instance, instead of
```
<compute_tendencies type="array(string)">NONE</compute_tendencies>
```
we can now just do
```
<compute_tendencies type="array(string)"></compute_tendencies>
```
or even
```
<compute_tendencies type="array(string)"/>
```
This removes the need in the code to handle the special case where the array has length 1, but contains the special value for "null". Instead, the code can simply check the array size, with size>0 implying valid values.

In order to do this I had to add stuff to our buildnml infrastructure. Namely, I had to create new user-defined YAML tags, signaling a sequence of bool/int/float/string values. I also needed to update EKAT, to get the correct version of the parser for these new tags.

E3SM-Autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Failure: Timed out waiting for job SCREAM_PullRequest_Autotester_Mappy to start: Total Wait = 1803
- Other jobs have been previously started - We must stop them...

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4250</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>ffa153104b9f1a3845c62847db22ab4fda239f7e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>b0039c2576ae9a67d22dd6c235876b9df651b35e</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4784</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>ffa153104b9f1a3845c62847db22ab4fda239f7e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>b0039c2576ae9a67d22dd6c235876b9df651b35e</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: bartgol/eamxx/buildnml-parsing-fix</li>
    <li>SHA: ffa153104b9f1a3845c62847db22ab4fda239f7e</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: bartgol
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: 1 or more Jobs FAILED

Note: Testing will normally be attempted again in approx. 2 Hrs. If a change to the PR source branch occurs, the testing will be attempted again on next available autotester run.

<details>
<summary>
  <b>Pull Request Auto Testing has FAILED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4250</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>ffa153104b9f1a3845c62847db22ab4fda239f7e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>b0039c2576ae9a67d22dd6c235876b9df651b35e</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4784</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>ffa153104b9f1a3845c62847db22ab4fda239f7e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>b0039c2576ae9a67d22dd6c235876b9df651b35e</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>


<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Mappy # 4250</b> (click to expand)
</summary>
<pre>
<code>
b'Finished SHAREDLIB_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 109.377132 seconds (PASS)'
b'Finished SHAREDLIB_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 124.764365 seconds (PASS)'
b'Starting MODEL_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished MODEL_BUILD for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 in 141.377132 seconds (PASS)'
b'Starting RUN for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished MODEL_BUILD for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 in 141.613619 seconds (PASS)'
b'Starting RUN for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 in 0.842294 seconds (PEND). [COMPLETED 1 of 6]'
b'Starting MODEL_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished RUN for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 in 0.828189 seconds (PEND). [COMPLETED 2 of 6]'
b'Finished MODEL_BUILD for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 in 175.441413 seconds (PASS)'
b'Starting RUN for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 in 0.813836 seconds (PEND). [COMPLETED 3 of 6]'
b'Finished MODEL_BUILD for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 175.956149 seconds (PASS)'
b'Starting RUN for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 0.831346 seconds (PEND). [COMPLETED 4 of 6]'
b'Finished MODEL_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 248.829061 seconds (PASS)'
b'Starting RUN for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 0.829351 seconds (PEND). [COMPLETED 5 of 6]'
b'Finished MODEL_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 294.016220 seconds (PASS)'
b'Starting RUN for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 1.533099 seconds (PEND). [COMPLETED 6 of 6]'
b'Waiting for tests to finish'
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_051911_dp8xqm'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_051911_dp8xqm'
b"test-scheduler took 500.3717975616455 seconds'"
b"+ errors='Waiting for tests to finish"
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_051911_dp8xqm'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_051911_dp8xqm'
b"test-scheduler took 500.3717975616455 seconds'"
b"+ V1_FAILURES_DETAILS+='Waiting for tests to finish"
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_051911_dp8xqm'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_051911_dp8xqm'
b"test-scheduler took 500.3717975616455 seconds'"
b'+ set +x'
b'######################################################'
b'FAILS DETECTED:'
b'  SCREAM STANDALONE TESTING FAILED!'
b"Build type full_debug failed at testing time. Here's a list of failed tests:"
b'227:ad_ut'
b'  SCREAM V1 TESTING FAILED!'
b'Waiting for tests to finish'
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_051911_dp8xqm'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_051911_dp8xqm'
b'FAIL SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_051911_dp8xqm'
b'test-scheduler took 500.3717975616455 seconds'
b'######################################################'
b"Build step 'Execute shell' marked build as failure"
b'Performing Post build task...'
b'Match found for : : True'
b'Logical operation result is TRUE'
b'Running script  : #!/bin/bash -le\r'
b'\r'
b'cd $WORKSPACE/${BUILD_ID}/\r'
b'\r'
b'./scream/components/eamxx/scripts/jenkins/jenkins_cleanup.sh'
b'[SCREAM_PullRequest_Autotester_Mappy] $ /bin/bash -le /tmp/jenkins9216336770178485109.sh'
b'POST BUILD TASK : SUCCESS'
b'END OF POST BUILD TASK : 0'
b'Sending e-mails to: lbertag@sandia.gov'
b'Finished: FAILURE'
b''
</code>
</pre>
</details>

<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Weaver # 4784</b> (click to expand)
</summary>
<pre>
<code>
b'CMake Error at /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/cmake/ctest_script.cmake:76 (message):'
b'  Test had fails'
b''
b''
b'==============================================================================='
b"Testing '\\''5e6a33432596b066129311ed5af46b7ba795b068'\\'' for test '\\''release'\\''"
b'==============================================================================='
b'RUN: taskset -c 104-155 sh -c \'\\\'\'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/release/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/release -DBUILD_NAME_MOD=release -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/cmake/ctest_script.cmake -DCTEST_SITE=weaver -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-fortran/4.6.0/gcc/11.3.0/openmpi/4.1.4/5ka6asw -DNetCDF_C_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-c/4.9.0/gcc/11.3.0/openmpi/4.1.4/mdd6fth -DCMAKE_BUILD_TYPE=Release -DEKAT_DISABLE_TPL_WARNINGS=\'\\\'\'\\\'\\\'\'\'\\\'\'ON\'\\\'\'\\\'\\\'\'\'\\\'\' -DCMAKE_CXX_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicxx -DCMAKE_C_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/release/data" \'\\\'\''
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/release'
b'==============================================================================='
b"Testing '\\''5e6a33432596b066129311ed5af46b7ba795b068'\\'' for test '\\''full_sp_debug'\\''"
b'==============================================================================='
b'RUN: taskset -c 52-103 sh -c \'\\\'\'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/full_sp_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/full_sp_debug -DBUILD_NAME_MOD=full_sp_debug -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/cmake/ctest_script.cmake -DCTEST_SITE=weaver -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-fortran/4.6.0/gcc/11.3.0/openmpi/4.1.4/5ka6asw -DNetCDF_C_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-c/4.9.0/gcc/11.3.0/openmpi/4.1.4/mdd6fth -DCMAKE_BUILD_TYPE=Debug -DEKAT_DEFAULT_BFB=True -DSCREAM_DOUBLE_PRECISION=False -DEKAT_DISABLE_TPL_WARNINGS=\'\\\'\'\\\'\\\'\'\'\\\'\'ON\'\\\'\'\\\'\\\'\'\'\\\'\' -DCMAKE_CXX_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicxx -DCMAKE_C_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_sp_debug/data" \'\\\'\''
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/full_sp_debug'
b'==============================================================================='
b"Testing '\\''5e6a33432596b066129311ed5af46b7ba795b068'\\'' for test '\\''full_debug'\\''"
b'==============================================================================='
b'RUN: taskset -c 0-51 sh -c \'\\\'\'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/full_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/full_debug -DBUILD_NAME_MOD=full_debug -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/cmake/ctest_script.cmake -DCTEST_SITE=weaver -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-fortran/4.6.0/gcc/11.3.0/openmpi/4.1.4/5ka6asw -DNetCDF_C_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-c/4.9.0/gcc/11.3.0/openmpi/4.1.4/mdd6fth -DCMAKE_BUILD_TYPE=Debug -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS=\'\\\'\'\\\'\\\'\'\'\\\'\'ON\'\\\'\'\\\'\\\'\'\'\\\'\' -DCMAKE_CXX_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicxx -DCMAKE_C_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_debug/data" \'\\\'\''
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx/ctest-build/full_debug'
b"Build type full_debug failed at testing time. Here'\\''s a list of failed tests:"
b'68:ad_ut'
b''
b"Build type full_sp_debug failed at testing time. Here'\\''s a list of failed tests:"
b'63:ad_ut'
b''
b"Build type release failed at testing time. Here'\\''s a list of failed tests:"
b'68:ad_ut'
b''
b'Error(s) occurred during test phase'
b'OVERALL STATUS: FAIL'
b'Starting analysis on weaver with cmd: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'RUN: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx'
b"weaver failed'"
b"+ errors='Build type full_debug failed at testing time. Here'\\''s a list of failed tests:"
b'68:ad_ut'
b''
b"Build type full_sp_debug failed at testing time. Here'\\''s a list of failed tests:"
b'63:ad_ut'
b''
b"Build type release failed at testing time. Here'\\''s a list of failed tests:"
b'68:ad_ut'
b''
b'Error(s) occurred during test phase'
b'OVERALL STATUS: FAIL'
b'Starting analysis on weaver with cmd: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'RUN: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx'
b"weaver failed'"
b"+ SA_FAILURES_DETAILS+='Build type full_debug failed at testing time. Here'\\''s a list of failed tests:"
b'68:ad_ut'
b''
b"Build type full_sp_debug failed at testing time. Here'\\''s a list of failed tests:"
b'63:ad_ut'
b''
b"Build type release failed at testing time. Here'\\''s a list of failed tests:"
b'68:ad_ut'
b''
b'Error(s) occurred during test phase'
b'OVERALL STATUS: FAIL'
b'Starting analysis on weaver with cmd: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'RUN: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx'
b"weaver failed'"
b'+ [[ 1 == 0 ]]'
b'+ [[ weaver == \\m\\a\\p\\p\\y ]]'
b'+ set +x'
b'######################################################'
b'FAILS DETECTED:'
b'  SCREAM STANDALONE TESTING FAILED!'
b"Build type full_debug failed at testing time. Here's a list of failed tests:"
b'68:ad_ut'
b''
b"Build type full_sp_debug failed at testing time. Here's a list of failed tests:"
b'63:ad_ut'
b''
b"Build type release failed at testing time. Here's a list of failed tests:"
b'68:ad_ut'
b''
b'Error(s) occurred during test phase'
b'OVERALL STATUS: FAIL'
b'Starting analysis on weaver with cmd: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'RUN: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4784/scream/components/eamxx'
b'weaver failed'
b'######################################################'
b"Build step 'Execute shell' marked build as failure"
b'Performing Post build task...'
b'Match found for : : True'
b'Logical operation result is TRUE'
b'Running script  : #!/bin/bash -le\r'
b'\r'
b'cd $WORKSPACE/${BUILD_ID}/\r'
b'\r'
b'./scream/components/eamxx/scripts/jenkins/jenkins_cleanup.sh'
b'[SCREAM_PullRequest_Autotester_Weaver] $ /bin/bash -le /tmp/jenkins6812059459889768856.sh'
b'POST BUILD TASK : SUCCESS'
b'END OF POST BUILD TASK : 0'
b'Sending e-mails to: lbertag@sandia.gov'
b'Finished: FAILURE'
b''
</code>
</pre>
</details>

jgfouca commented on components/eamxx/cime_config/eamxx_buildnml.py:
```diff
@@ -238,15 +246,16 @@ def evaluate_selectors(element, case, ez_selectors):
             evaluate_selectors(child, case, ez_selectors)
         else:
+            child_name = child.tag
+            child.text = None if child.text is None else child.text.strip(' \n')
```
Text should never be `None` now, right?

bartgol replied:
If there is a parameter like `<my_param/>` (open and closed in one set of brackets), then the text will indeed be set to None by the XML parser.

jgfouca replied:
@bartgol 

That's is unfortunate.

I thought `<x/>` was just semantic sugar for `<x></x>`. Are you saying the former gives text=None and the latter gives text=""?

bartgol replied:
I was about to say "yes", then I checked with this mini script
```python
import xml.etree.ElementTree as ET


s = """ 
<tree>
    <a/>
    <b></b>
</tree>
"""
xml = ET.fromstring(s)

for a in xml:
    print(f"{a.tag} = {a.text}")
```
which printed
```bash
a = None
b = None
```
so now I'm not sure anymore...

jgfouca commented on components/eamxx/cime_config/eamxx_buildnml.py:
```diff
@@ -745,6 +754,8 @@ def create_input_data_list_file(caseroot):
 ###############################################################################
 def do_cime_vars_on_yaml_output_files(case, caseroot):
 ###############################################################################
+    from yaml_utils import array_constructor
```
I'd prefer this be at the top with the other imports.

bartgol replied:
ok

jgfouca commented on components/eamxx/cime_config/eamxx_buildnml_impl.py:
```diff
@@ -190,6 +190,8 @@ def has_child (root,name):
 ###############################################################################
 def refine_type(entry, force_type=None):
 ###############################################################################
+    from yaml_utils import make_array
```
Ditto

jgfouca commented on components/eamxx/cime_config/yaml_utils.py:
```diff
@@ -0,0 +1,68 @@
+        return Strings(vals)
+    else:
+        raise ValueError (f"Unsupported element type '{etype}' for arrays.")
+###############################################################################
```
Move this second hash line directly below `def make_array`

jgfouca reviewed (changes requested):
Some minor nits. Looks good overall.

E3SM-Autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4260</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>95c3357f72b6f4259415233f5a4c62810ca6924f</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>f449f922ec2ec0a14a84490ec6a52b9adbfb9c2f</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4794</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>95c3357f72b6f4259415233f5a4c62810ca6924f</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>f449f922ec2ec0a14a84490ec6a52b9adbfb9c2f</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: bartgol/eamxx/buildnml-parsing-fix</li>
    <li>SHA: 95c3357f72b6f4259415233f5a4c62810ca6924f</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: bartgol
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: 1 or more Jobs FAILED

Note: Testing will normally be attempted again in approx. 2 Hrs. If a change to the PR source branch occurs, the testing will be attempted again on next available autotester run.

<details>
<summary>
  <b>Pull Request Auto Testing has FAILED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4260</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>95c3357f72b6f4259415233f5a4c62810ca6924f</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>f449f922ec2ec0a14a84490ec6a52b9adbfb9c2f</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4794</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>95c3357f72b6f4259415233f5a4c62810ca6924f</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>f449f922ec2ec0a14a84490ec6a52b9adbfb9c2f</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>


<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Mappy # 4260</b> (click to expand)
</summary>
<pre>
<code>
b'Starting MODEL_BUILD for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 with 16 procs'
b'Finished SHAREDLIB_BUILD for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 55.768809 seconds (PASS)'
b'Starting MODEL_BUILD for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished SHAREDLIB_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 115.642507 seconds (PASS)'
b'Finished SHAREDLIB_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 132.218580 seconds (PASS)'
b'Starting MODEL_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished MODEL_BUILD for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 in 147.244092 seconds (PASS)'
b'Starting RUN for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished MODEL_BUILD for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 in 147.933571 seconds (PASS)'
b'Starting RUN for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 in 0.829004 seconds (PEND). [COMPLETED 1 of 6]'
b'Starting MODEL_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished RUN for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 in 0.796925 seconds (PEND). [COMPLETED 2 of 6]'
b'Finished MODEL_BUILD for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 in 192.256225 seconds (PASS)'
b'Starting RUN for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 in 0.878062 seconds (PEND). [COMPLETED 3 of 6]'
b'Finished MODEL_BUILD for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 209.450966 seconds (PASS)'
b'Starting RUN for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 3.477554 seconds (PEND). [COMPLETED 4 of 6]'
b'Finished MODEL_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 294.170031 seconds (PASS)'
b'Starting RUN for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 0.950353 seconds (PEND). [COMPLETED 5 of 6]'
b'Finished MODEL_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 336.035522 seconds (PASS)'
b'Starting RUN for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 1.050701 seconds (PEND). [COMPLETED 6 of 6]'
b'Waiting for tests to finish'
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_141113_pi0sx3'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_141113_pi0sx3'
b"test-scheduler took 546.5255007743835 seconds'"
b"+ errors='Waiting for tests to finish"
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_141113_pi0sx3'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_141113_pi0sx3'
b"test-scheduler took 546.5255007743835 seconds'"
b"+ V1_FAILURES_DETAILS+='Waiting for tests to finish"
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_141113_pi0sx3'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_141113_pi0sx3'
b"test-scheduler took 546.5255007743835 seconds'"
b'+ set +x'
b'######################################################'
b'FAILS DETECTED:'
b'  SCREAM V1 TESTING FAILED!'
b'Waiting for tests to finish'
b'FAIL ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'FAIL ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230824_141113_pi0sx3'
b'FAIL ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase RUN)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230824_141113_pi0sx3'
b'PASS SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 RUN'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230824_141113_pi0sx3'
b'test-scheduler took 546.5255007743835 seconds'
b'######################################################'
b"Build step 'Execute shell' marked build as failure"
b'Performing Post build task...'
b'Match found for : : True'
b'Logical operation result is TRUE'
b'Running script  : #!/bin/bash -le\r'
b'\r'
b'cd $WORKSPACE/${BUILD_ID}/\r'
b'\r'
b'./scream/components/eamxx/scripts/jenkins/jenkins_cleanup.sh'
b'[SCREAM_PullRequest_Autotester_Mappy] $ /bin/bash -le /tmp/jenkins17574360401769934651.sh'
b'POST BUILD TASK : SUCCESS'
b'END OF POST BUILD TASK : 0'
b'Sending e-mails to: lbertag@sandia.gov'
b'Finished: FAILURE'
b''
</code>
</pre>
</details>

<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Weaver # 4794</b> (click to expand)
</summary>
<pre>
<code>
b'Running as SYSTEM'
b'Building remotely on weaver in workspace /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver'
b'The recommended git tool is: NONE'
b'using credential cf1815d3-62c3-4157-a38d-0ad235093618'
b'Wiping out workspace first.'
b'Cloning the remote Git repository'
b'Honoring refspec on initial clone'
b'Cloning repository https://github.com/e3sm-project/scream.git'
b' > /home/projects/ppc64le/git/2.10.1/bin/git init /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4794/scream # timeout=10'
b'Fetching upstream changes from https://github.com/e3sm-project/scream.git'
b' > /home/projects/ppc64le/git/2.10.1/bin/git --version # timeout=10'
b" > git --version # 'git version 2.10.1'"
b'using GIT_SSH to set credentials jgfouca github key'
b'Verifying host key using manually-configured host key entries'
b'Setting http proxy: proxy.sandia.gov:80'
b' > /home/projects/ppc64le/git/2.10.1/bin/git fetch --tags --progress -- https://github.com/e3sm-project/scream.git +refs/heads/master:refs/remotes/origin/master +refs/pull/2468/head:refs/heads/pr/2468 # timeout=30'
b"ERROR: Error cloning remote repo 'origin'"
b'hudson.plugins.git.GitException: Command "/home/projects/ppc64le/git/2.10.1/bin/git fetch --tags --progress -- https://github.com/e3sm-project/scream.git +refs/heads/master:refs/remotes/origin/master +refs/pull/2468/head:refs/heads/pr/2468" returned status code 128:'
b'stdout: '
b'stderr: remote: Internal Server Error'
b"fatal: unable to access 'https://github.com/e3sm-project/scream.git/': The requested URL returned error: 500"
b''
b'\tat org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:2732)'
b'\tat org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandWithCredentials(CliGitAPIImpl.java:2109)'
b'\tat org.jenkinsci.plugins.gitclient.CliGitAPIImpl$1.execute(CliGitAPIImpl.java:623)'
b'\tat org.jenkinsci.plugins.gitclient.CliGitAPIImpl$2.execute(CliGitAPIImpl.java:850)'
b'\tat org.jenkinsci.plugins.gitclient.RemoteGitImpl$CommandInvocationHandler$GitCommandMasterToSlaveCallable.call(RemoteGitImpl.java:158)'
b'\tat org.jenkinsci.plugins.gitclient.RemoteGitImpl$CommandInvocationHandler$GitCommandMasterToSlaveCallable.call(RemoteGitImpl.java:151)'
b'\tat hudson.remoting.UserRequest.perform(UserRequest.java:211)'
b'\tat hudson.remoting.UserRequest.perform(UserRequest.java:54)'
b'\tat hudson.remoting.Request$2.run(Request.java:377)'
b'\tat hudson.remoting.InterceptingExecutorService.lambda$wrap$0(InterceptingExecutorService.java:78)'
b'\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)'
b'\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)'
b'\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)'
b'\tat java.base/java.lang.Thread.run(Thread.java:829)'
b'\tSuppressed: hudson.remoting.Channel$CallSiteStackTrace: Remote call to weaver'
b'\t\tat hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1784)'
b'\t\tat hudson.remoting.UserRequest$ExceptionResponse.retrieve(UserRequest.java:356)'
b'\t\tat hudson.remoting.Channel.call(Channel.java:1000)'
b'\t\tat org.jenkinsci.plugins.gitclient.RemoteGitImpl$CommandInvocationHandler.execute(RemoteGitImpl.java:143)'
b'\t\tat jdk.internal.reflect.GeneratedMethodAccessor293.invoke(Unknown Source)'
b'\t\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)'
b'\t\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)'
b'\t\tat org.jenkinsci.plugins.gitclient.RemoteGitImpl$CommandInvocationHandler.invoke(RemoteGitImpl.java:129)'
b'\t\tat com.sun.proxy.$Proxy92.execute(Unknown Source)'
b'\t\tat hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1226)'
b'\t\tat hudson.plugins.git.GitSCM.checkout(GitSCM.java:1309)'
b'\t\tat hudson.scm.SCM.checkout(SCM.java:540)'
b'\t\tat hudson.model.AbstractProject.checkout(AbstractProject.java:1240)'
b'\t\tat hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:649)'
b'\t\tat jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:85)'
b'\t\tat hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:521)'
b'\t\tat hudson.model.Run.execute(Run.java:1900)'
b'\t\tat hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)'
b'\t\tat hudson.model.ResourceController.execute(ResourceController.java:101)'
b'\t\tat hudson.model.Executor.run(Executor.java:442)'
b"ERROR: Error cloning remote repo 'origin'"
b'Performing Post build task...'
b'Match found for : : True'
b'Logical operation result is TRUE'
b'Running script  : #!/bin/bash -le\r'
b'\r'
b'cd $WORKSPACE/${BUILD_ID}/\r'
b'\r'
b'./scream/components/eamxx/scripts/jenkins/jenkins_cleanup.sh'
b'[SCREAM_PullRequest_Autotester_Weaver] $ /bin/bash -le /tmp/jenkins1611041671590997045.sh'
b'/tmp/jenkins1611041671590997045.sh: line 5: ./scream/components/eamxx/scripts/jenkins/jenkins_cleanup.sh: No such file or directory'
b'POST BUILD TASK : FAILURE'
b'END OF POST BUILD TASK : 0'
b'Sending e-mails to: lbertag@sandia.gov'
b'Finished: FAILURE'
b''
</code>
</pre>
</details>

E3SM-Autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4266</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>0dc3c6b4d6904e7051a922db94475b63224606c0</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>affda5db6534a0c6951e8045c54856173af9dd8c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4800</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>0dc3c6b4d6904e7051a922db94475b63224606c0</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>affda5db6534a0c6951e8045c54856173af9dd8c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: bartgol/eamxx/buildnml-parsing-fix</li>
    <li>SHA: 0dc3c6b4d6904e7051a922db94475b63224606c0</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: bartgol
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4266</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>0dc3c6b4d6904e7051a922db94475b63224606c0</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>affda5db6534a0c6951e8045c54856173af9dd8c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4800</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>bugfix;CIME</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2468</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>0dc3c6b4d6904e7051a922db94475b63224606c0</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>affda5db6534a0c6951e8045c54856173af9dd8c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>

E3SM-Autotester commented:
Status Flag 'Pre-Merge Inspection' - - This Pull Request Requires Inspection... The code must be inspected by a member of the Team before Testing/Merging
           THE LAST COMMIT TO THIS PULL REQUEST HAS NOT BEEN REVIEWED YET!

E3SM-Autotester commented:
All Jobs Finished; status = PASSED, However Inspection must be performed before merge can occur...

E3SM-Autotester commented:
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ jgfouca ]!

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - AutoMerge IS ENABLED, but the Label AT: AUTOMERGE is not set.  Either set Label AT: AUTOMERGE or manually merge the PR...
